chartsLibrary/SolidSparklineChart.tsx

```diff
@@ -0,0 +1,13 @@
+import { onMount } from "solid-js";
+import { AreaChart } from "@carbon/charts";
+import { ChartConfig, AreaChartOptions } from "@carbon/charts/interfaces";
+
+let divRef: HTMLDivElement = (<div></div>) as HTMLDivElement;
+
+export default function SolidLineChart(props: ChartConfig<AreaChartOptions>) {
```

Comment:
This function is AreaLineChart... does not really matter. But please correct it

Reply:
ok